browser/src/control/Control.StatusBar.js

```diff
@@ -248,10 +248,10 @@ class StatusBar extends JSDialog.Toolbar {
 			{type: 'customtoolitem',  id: 'next', command: 'next', text: _UNO('.uno:PageDown', 'text')},
 			{type: 'separator', id: 'prevnextbreak', orientation: 'vertical'},
 		].concat(window.mode.isTablet() ? [] : [
-			{type: 'customtoolitem',  id: 'zoomreset', command: 'zoomreset', text: _('Reset zoom')},
-			{type: 'customtoolitem',  id: 'zoomout', command: 'zoomout', text: _UNO('.uno:ZoomMinus')},
+			{type: 'customtoolitem',  id: 'zoomreset', command: 'zoomreset', text: _('Reset zoom'), image: 'images/zoomreset.svg'},
+			{type: 'customtoolitem',  id: 'zoomout', command: 'zoomout', text: _UNO('.uno:ZoomMinus'), image: 'images/minus.svg'},
 			{type: 'menubutton', id: 'zoom', text: '100', selected: 'zoom100', menu: this._generateZoomItems(), image: false},
-			{type: 'customtoolitem',  id: 'zoomin', command: 'zoomin', text: _UNO('.uno:ZoomPlus')}
+			{type: 'customtoolitem',  id: 'zoomin', command: 'zoomin', text: _UNO('.uno:ZoomPlus'), image: 'images/plus.svg'}
```

Comment:
just 'plus' doesn't work? or maybe it was other property name like: "img:".
Paths are generated by the toolitem widget, would be good to avoid hardcoding here

Reply:
yes I did a mistake, it should be img instead, thanks